meta-balena-common/recipes-core/balena-rollback/files/rollback-health

```diff
@@ -124,6 +125,84 @@ secureboot_apply_dbx() {
 	rm -rf "${TMPDIR}"
 }
 
+secureboot_update_policy() {
```

Comment:
As this is EFI specific I would rename to `efi_update_policy` and add it only for device types that define `efi` as a machine feature.